addclear.js

```diff
@@ -64,15 +66,14 @@
 					me = this,
 					options = this.options;
 
-			$this.wrap("<span style='position:relative;' class='add-clear-span'></span>");
```

Comment:
Missed this before I merged but `style='position:relative;'` shouldn't have been removed.